server/src/routes/api/v1/votesRouter.js

```diff
@@ -0,0 +1,58 @@
+import express from 'express'
+import { Vote } from '../../../models/index.js'
+import VoteSerializer from '../../../../serializers/VoteSerializer.js'
+import { Review } from '../../../models/index.js'
+
+const votesRouter = new express.Router()
+
+votesRouter.get('/:reviewId', async (req, res) => {
+  try {
+    const { reviewId } = req.params
+    const totalCount = await VoteSerializer.getTotal(reviewId)
+    return res.status(201).json({ totalCount })
```

Comment:
Being a GET path now the status code should be 200

We should also consider grabbing this data when the Show page with reviews is loading. Right now the fetch request for the total vote count is on EVERY single review tile, meaning each review has to make a fetch for its data, which is not ideal. When fetching for the infor for the show page, and it's reviews, we can also send along the current vote data

This VoteSerializer can be added to the review serializer when you are grabbing all the reviews for the service